metrics/README.md

```diff
@@ -19,3 +19,21 @@ Once you have these things, run
 
 which should generate new metrics and place them into `fontMetrics.js`. You're
 done!
+
+### OS X Notes
+
+Install the JSON perl module:
+    
+    sudo cpan JSON
+
+Install fontforge with python extensions (this may install python as well):
+
+    brew install fontforge --enable-pyextension
+    
+If you get a "Segmentation fault: 11" when running `make metrics` you'll have
```

Comment:
The system version of python is 2.7.5.  I tried `#!/usr/bin/env python2`, but it still used 2.7.5 instead of 2.7.9.